hr_timesheet_report_not_valued/report/hr_timesheet_report.xml

```diff
@@ -0,0 +1,28 @@
+<odoo>
+    <template id="report_timesheet_not_valued_1" inherit_id="hr_timesheet.timesheet_table">
+        <xpath expr="//th[@class='text-right']" position="attributes">
+            <attribute name="t-if">not not_valued</attribute>
+        </xpath>
+        <xpath expr="//tbody/tr[1]/td[@class='text-right']" position="attributes">
+            <attribute name="t-if">not not_valued</attribute>
+        </xpath>
+        <xpath expr="//tbody/tr[2]/td[@class='text-right']/strong[2]" position="attributes">
+            <attribute name="t-if">not not_valued</attribute>
+        </xpath>
+    </template>
+
+    <template id="report_timesheet_not_valued">
+        <t t-set="not_valued" t-value="True"/>
+        <t t-call="hr_timesheet.report_timesheet"/>
+    </template>
+
+    <record id="timesheet_not_valued_report" model="ir.actions.report">
+        <field name="name">Timesheet Entries Not Valued</field>
+        <field name="model">account.analytic.line</field>
+        <field name="report_type">qweb-pdf</field>
+        <field name="report_name">hr_timesheet_report_not_valued.report_timesheet_not_valued</field>
+        <field name="report_file">report_timesheet</field>
+        <field name="binding_model_id" ref="account.analytic.line" />
```

Comment:
This is wrong and fires an error when installing (`ref` attribute in XML data shoud point to a valid XML identifier). Try this:
```suggestion
        <field
            name="binding_model_id"
            ref="hr_timesheet.model_account_analytic_line"
        />
```